src/utils/navigation.ts

```diff
@@ -104,16 +109,16 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.OVERSEAS_ENTITY_REVIEW_URL,
     nextPage: []
   },
-  [config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL]: {
-    currentPage: config.BENEFICIAL_OWNER_STATEMENTS_PAGE,
-    previousPage: () => config.OVERSEAS_ENTITY_REVIEW_URL,
-    nextPage: [config.UPDATE_REGISTRABLE_BENEFICIAL_OWNER_URL]
-  },
   [config.UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_URL]: {
     currentPage: config.UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_PAGE,
     previousPage: () => config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE,
     nextPage: [config.UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL]
   },
+  [config.UPDATE_BENEFICIAL_OWNER_STATEMENTS_URL]: {
```

Comment:
Can you update the `navigation.spec.ts` tests please?

Reply:
Test added